PolyEngine/Engine/Src/DebugDrawSystem.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include "Vector.hpp"
+
+namespace Poly
+{
+	class World;
+
+	namespace DebugDrawSystem
+	{
+		void ENGINE_DLLEXPORT DebugRenderingUpdatePhase(World*);
+
+		/// Emitters of elementary shapes
+		void EmitLine(World* world, Vector begin, Vector end);
```

Comment:
Draw would be better than Emit. What do you think? Is there any reason you went with Emit?

Reply:
I put emit in this method's name because it only pushes a primitive (line) into vertices-to-be-rendered queue. It kinda doesn't draw anything.
But most consecutive methods `Emit*` actually do create shapes in space which I'd call drawing. So to keep method names consistent I agree, `Draw*` is better.